deshift/_src/spectra.py

```diff
@@ -15,6 +15,7 @@ def make_superquantile_spectrum(batch_size: int, tail_prob: float):
       spectrum
         a sorted vector of ``n`` weights on each training example.
     """
+    assert type(batch_size) == int
```

Comment:
The standard syntax is ``assert isinstance(batch_size, int)`` (not very important)